EcoSoapBank/Model/Pickup.swift

```diff
@@ -69,7 +70,7 @@ extension Pickup: Decodable {
         let status: Status
         let readyDate: Date
         let pickupDate: Date?
-        let notes: String
+        let notes: String?
```

Comment:
At first I was thinking we would just provide an empty string if there are no notes, but I suppose it does make sense to just not provide them if there are none. Might also make sense to add some logic to set this value to `nil` if it's empty. I'm not sure where we should put that logic though.

Reply:
Hmm, I'll think about that. I can probably do that in the custom decoder I forgot to write. If it's there to decode, set it, otherwise, set it to nil?